tests/utils/test_proto_json_utils.py

```diff
@@ -85,6 +88,7 @@ def assert_result(result, expected_result):
         assert result.keys() == expected_result.keys()
         for key in result:
             assert (result[key] == expected_result[key]).all()
+            assert result[key].dtype == expected_result[key].dtype
```

Comment:
The windows test is failing here because python interprets int as 32 bits be default in windows
https://stackoverflow.com/questions/22513445/python-handles-long-ints-differently-on-windows-and-unix.
Arggg.

Reply:
Hmm thanks, I'll look into that.